lib/tasks/spin_up.rake

```diff
@@ -32,28 +42,26 @@ class SpinUpHelper
   def setup_secrets_config
     filename = "config/secrets.yml"
     yaml = YAML.load_file(filename)
-    yaml["development"]["secret_key_base"] = SecureRandom.hex(128)
-    yaml["test"]["secret_key_base"] = SecureRandom.hex(128)
+    yaml["development"]["secret_key_base"] = SecureRandom.hex(64)
+    yaml["test"]["secret_key_base"] = SecureRandom.hex(64)
```

Comment:
I was using SecureRandom.hex wrong. It outputs twice the number you put in, so 64 will return 128 characters.